src/services/users/reset-password/index.js

```diff
@@ -0,0 +1,19 @@
+const { 
```

Comment:
file looks good, we can formulate a test for this in a file called `index.test.js` 

the main thing we'll be testing is two scenarios
1. it throwing the required error when token is invalid
2. the database update being done successfully

Kindly check out some new utils functions that have been added to make the creation of a user easy, this pr has a few pointers on the format now: https://github.com/tbf-challenge/teamwork-be/pull/201/files

Please feel free to reach out about anything I've mentioned